src/main/java/de/komoot/photon/PhotonDoc.java

```diff
@@ -75,6 +75,9 @@ public static PhotonDoc create(long placeId, String osmType, long osmId, Map<Str
 	}
 
 	public boolean isUsefulForIndex() {
+            
+                if(centroid == null) return false;
+                    
```

Comment:
OSM data always has a geometry, but I realized that many of the csv datasets that I had to deal with had some entries that were lacking the geometry information. I therefore added this check to make sure that those documents do not get added to the index (since they are not really useful for geocoding if they don't have the lat lon values).